frontend/src/pages/org/browser-profiles-list.ts

```diff
@@ -101,7 +102,18 @@ export class BrowserProfilesList extends LiteElement {
       >
         <div class="grid grid-cols-8 gap-3 md:gap-5" role="row">
           <div class="col-span-8 md:col-span-3 p-2" role="cell">
-            <div class="font-medium mb-1">${data.name}</div>
+            <div class="font-medium text-sm">
+              <span>${data.name}</span>
+              ${when(
+                data.resource && data.resource.replicas.length > 0,
+                () => html` <sl-tooltip content=${msg("Profile replicated")}>
+                  <sl-icon
+                    name="cloud-check"
```

Comment:
Heres a suggestion for changing to [`clouds`](https://icons.getbootstrap.com/icons/clouds/) :)

```suggestion
                    name="clouds"
```